docs/_blog/index.md

```diff
@@ -22,7 +22,7 @@ exclude_from_loop: true
 
 <div class="news_cards">
 
-  {% assign sorted = (site.blog | sort: 'date') | reverse %}
+  {% assign sorted = site.blog | sort: 'date' | reverse %}
```

Comment:
revert unless its related to this PR

Reply:
Yes this is due to the change from redcarpet to kramdown